module/scripts/project/thumbnail-renderer.js

```diff
@@ -61,7 +61,7 @@ class ThumbnailReconRenderer extends ReconCellRenderer {
         .attr("target", "_blank")
         .appendTo(divContentRecon);
 
-      var bareFileName = match.name.substr('File:'.length).replaceAll(' ', '_');
+      var bareFileName = match.name.includes('File:') ? match.name.substr('File:'.length).replaceAll(' ', '_') : match.name.replaceAll(' ', '_');
```

Comment:
```suggestion
      var bareFileName = match.name.startsWith('File:') ? match.name.substr('File:'.length).replaceAll(' ', '_') : match.name.replaceAll(' ', '_');
```

Just in case `File:` is contained somewhere else in the name…

Reply:
@wetneb Thank you. I have committed the proposed changes.